server/pkg/controller/replication3.go

```diff
@@ -85,12 +85,19 @@ func (c *ReplicationController3) StartReplication() error {
 		return fmt.Errorf("v3 replication can currently only run when the primary hot data center is Backblaze. Instead, it was %s", hotDC)
 	}
 
+	// If bypassWorker is true, we set c.workerURL to "", which makes downloadFromB2ViaWorker
+	// bypass the download worker. c.workerURL should not be empty otherwise
+	bypassWorker := viper.GetBool("replication.bypass-worker")
```

Comment:
On reading the existing code again, I'm wondering if we should just make the worker-url optional. @nicegamer7 what are your thoughts on this?  There is extra cognitive and complexity cost of adding a new option, so if we just instead make the worker-url optional and use its absence as a signal for direct replication.

Reply:
I agree that adding a new option adds unneeded complexity, especially since it seems there aren't any automated tests to ensure this will stay working as intended. For that reason I think it'd be better to just make worker-url optional.

The only reason I didn't do that in the first place was to prevent changing the server's existing behaviour, since I wasn't sure if that'd be welcome.

I'll get this changed.

Reply:
> The only reason I didn't do that in the first place was to prevent changing the server's existing behaviour, since I wasn't sure if that'd be welcome.

Yes, I understood you'd intentionally tried to minimize the diff. Thank you!